### DISCUSSION TITLE
Not able to update bootstrapconfig

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx-operator#525] [Q&A] [answered]
Hi,
I tried to update the bootstrap config using kubectl apply, edit and patch but none of these worked, all gives 
`Error from server (bootstrap config cannot be updated, old bootstrap config: `

Any way to update bootstrapconfig without deleting emqx resource or restarting the pods?
Thanks in advance...

**Comment by Rory-Z** [comment, marked as answer]
Hi, @alexa22013  `bootstrapConfig` is designed not to allow updates, if you need to update the configuration of a running emqx, you can use the EMQX Dashboard
